src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx

```diff
@@ -118,22 +116,23 @@ const VideoInputBackgroundBlurControl: React.FC<Props> = ({
             await meetingManager.selectVideoInputDevice(option.deviceId);
           }
         }}
-      />
+      >
+        <span>{option.label}</span>
+      </PopOverItem>
     ));
     if (isBackgroundBlurSupported) {
       const videoTransformOptions: ReactNode = (
         <PopOverItem
           key="videoinput"
-          children={
-            <>
-              {isLoading && <Spinner width="1.5rem" height="1.5rem" />}
-              {backgroundBlurLabel}
-            </>
-          }
           checked={isVideoTransformDevice(activeVideoDevice)}
           disabled={isLoading}
           onClick={toggleBackgroundBlur}
-        />
+        >
+          <>
```

Comment:
Yeah, we can remove this. Will update later.

Reply:
Confirmed we can not remove this.

Reply:
Can we put a comment on why we can't remove it?

Reply:
If we remove it will be a type error.

```
Type 'string | false | Element' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.
Type 'string' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'
```

```tsx
children?: React.ReactElement<any> | React.ReactElement<any>[];
```